content/api/cli.md

```diff
@@ -195,38 +195,38 @@ These options allow webpack to display various stats and style them differently
 | Parameter               | Explanation                                                        | Type    |
 |-------------------------|--------------------------------------------------------------------|---------|
 | --color, --colors       | Enables/Disables colors on the console [default: (supports-color)] | boolean |
```

Comment:
I think it's still in [use](https://github.com/webpack/webpack/blob/f18989c712894bba13ebfb6dfb3f50fc44b0d12a/bin/webpack.js#L279).